### PR TITLE
Add PnP Typescript Preset

### PR DESCRIPTION
**Pull request by crubier**
Fixes this bug 

https://github.com/storybookjs/storybook/issues/6905

See also

https://github.com/storybookjs/storybook/issues/6899

### CODE REVIEW

File: packages/preset-yarn-pnp-ts/.storybook/presets.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = [
+  {
+    name: '@storybook/preset-yan-pnp-ts',
```
Comment (joeycozza) [inline packages/preset-yarn-pnp-ts/.storybook/presets.ts]: was this supposed to be 
```suggestion
    name: '@storybook/preset-yarn-pnp-ts',
```

**Comment by ndelangen** [comment]
@igor-dv is there the possibility that you finish this PR yourself? Would be great to have this merged some time soon!

**Comment by igor-dv** [comment]
Yeah, I have plans to do this at the weekend

**Comment by igor-dv** [comment]
merged to a different branch so I could working on it

**Pull request merged**